client/pages/Compose.js

```diff
@@ -137,18 +137,21 @@ export default function () {
           <p>{errMsg.length == 0 ? " " : errMsg}</p>
           <div class="buttons">
             <CustomButton
+              className="button is-success"
```

Comment:
The CustomButton component does not take `className` props. This is an intentional "feature" to improve the consistency of the application - all instances of CustomButton look the same.

And we want them to all look the same!